verticapy/tests/base.py

```diff
@@ -20,6 +20,7 @@
 import inspect
 import getpass
 import vertica_python
+import verticapy
```

Comment:
I think this import is redundant. Let's avoid redundant imports.

Reply:
verticapy != vertica_python

Reply:
I know :), but `importing verticapy` looks redundant while the content of the file is not changed. Have you tried to run the tests without this import?

Reply:
Done;